site/about/style-guide.qmd

```diff
@@ -0,0 +1,230 @@
+---
+title: "ValidMind Style Guide"
+date: last-modified
+---
+
+A style guide helps create distinct yet unified communication across all areas of a product experience, from in-app interactions to technical documentation and blog posts. 
+
+## Goals
+At ValidMind, we value **transparency and accessibility**—we aim to speak simply and effectively. We also believe in **creating community**, by presenting information in a manner that encourages collaboration and feedback from users old and new. 
+
+Our processes reflect a **holistic journey**—accommodating the needs of users begins in the design phase, documentation supplements instead of replaces intuitive or engaging and guided user experiences, and publications should help our audience understand value and maximize their potential with ValidMind. 
+
+> The following guidelines are meant to reflect these above principles, and ensure that all of our communications adhere to our vision. 
```

Comment:
DISCUSS: This feels like a misuse of the quote markup. This isn't a quote, it's us saying what this is about. 

Maybe .callout-info would be more appropriate here? Or just plain italics?

Reply:
I just wanted some visual distinction. Can you suggest a change to a callout?

::: {.callout-tip appearance="minimal"}
The following guidelines are meant to reflect these above principles, and ensure that all of our communications adhere to our vision. 
:::

Reply:
Yup, a callout like the one you're suggesting will work. 👍